pkg/result/results.go

```diff
@@ -513,6 +515,7 @@ func (r *Summary) PrettyPrint() string {
 	}
 
 	sb.WriteString("------------------------------------------\n")
+	sb.WriteString(fmt.Sprintf("Average duration of %.2f seconds (min: %.2f max: %.2f std: %.2f)\n", r.Duration.Mean, r.Duration.Min, r.Duration.Max, r.Duration.SD))
```

Comment:
Should we only print this in HP/TTK mode?

Reply:
I guess it should be constant with SD of 0 in non HP/TTK mode? It's fien for now imo...